test/libsyncengine/jobs/network/testnetworkjobs.cpp

```diff
@@ -719,27 +719,28 @@ void TestNetworkJobs::testDownload() {
 }
 
 void TestNetworkJobs::testDownloadHasEnoughSpace() {
-    if (!testhelpers::isRunningOnCI() && testhelpers::isExtendedTest(false)) return;
-
-    // Only run on CI because it requires a small partition to be set up)
-    const SyncPath smallPartitionPath = testhelpers::TestVariables().local8MoPartitionPath;
-    if (smallPartitionPath.empty()) return;
+    if (testhelpers::isRunningOnCI() && testhelpers::isExtendedTest(false)) {
```

Comment:
```suggestion
    if (!testhelpers::isRunningOnCI() || testhelpers::isExtendedTest(true)) return;
```

In most places in the code, we usually prefer early return statement to improve readability (less brackets)

Reply:
[f0f4162](https://github.com/Infomaniak/desktop-kDrive/pull/1243/commits/f0f4162d971c327daadc086d307411ff9b99a4c1)